mobidam-sst-management-backend/src/main/java/de/muenchen/mobidam/rest/SchnittstelleController.java

```diff
@@ -52,6 +53,15 @@ public ResponseEntity<Iterable<SchnittstelleDTO>> getAll() {
         return new ResponseEntity<>(schnittstelleService.getAll(), HttpStatus.OK);
     }
 
+    @Operation(summary = "Get status of schnittstelle")
+    @GetMapping("{id}/status")
```

Comment:
Bin mir unsicher, aber ist nicht die Konvention, dass die Path-Parameter als letztes kommen?
```suggestion
    @GetMapping("status/{id}")
```

Reply:
@mirrodi Ich habe dazu keine Konvention gefunden. Wenn Du mir einen Link nennen kannst, schau ich mal drüber.

Reply:
Hab Query's und Path-Parameters verwechselt anscheinend. Müsste dann so passen wie es ist